client-side/src/components/profileComponents/ProfileActions.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Tooltip, Button } from '@mui/material';
+import { TOOLTIP_TEXTS, BUTTON_LABELS } from '../../constants/profileConstants.js';
+
+export default function ProfileActions({ handleSave, handleClose, handleDelete }) {
```

Comment:
please add default values for each prop that not marked as required, also please add prop-type definition